disco_aws_automation/disco_elb.py

```diff
@@ -367,7 +367,7 @@ def _describe_instance_health(self, elb_id, instance_ids=None):
         return instances["InstanceStates"]
 
     def wait_for_instance_health_state(self, hostclass, testing=False, instance_ids=None, state="InService",
-                                       timeout=180):
+                                       timeout=600):
```

Comment:
Why does it require 10 mins to spin up instances? Do we know the root cause of the failure?